### DISCUSSION TITLE
Patch method not included in access-control-allow-methods

### DISCUSSION THREAD

**Discussion by Barkil** [pocketbase/pocketbase#692] [Q&A] [answered]
I was tinkering around, and noticed I could not get the update to work in the JS SDK.  The create, delete, get all work perfected.

I have my actions wide open on rules.
![image](https://user-images.githubusercontent.com/114616599/193438322-ba8a2830-e81f-4808-b699-c9ead11884fb.png)

And I noticed Patch isn't in the preflight options in the access-control-allow-methods
![image](https://user-images.githubusercontent.com/114616599/193438263-db877f53-90a0-44cd-ab2f-0f93891b0283.png)

Which results in the following console error.
![image](https://user-images.githubusercontent.com/114616599/193438399-976bb601-96ed-4f92-bcf1-de93a42d81ba.png)

Looking for guidance on this.

**Comment by ganigeorgiev** [comment, marked as answer]
That's strange. `PATCH` is already enabled:

![cors screenshot](https://i.imgur.com/iUpaEXS.png)

You could find the code in https://github.com/pocketbase/pocketbase/blob/master/cmd/serve.go#L55

Do you have a proxy, ssh tunneling or some browser extension that could change the response headers?

**Reply by Barkil** [reply]
_Do you have a proxy, ssh tunneling or some browser extension that could change the response headers?_

Totally had an extension for cors I completely forgot about. Thanks a lot @ganigeorgiev.

**Comment by marckrenn** [comment]
Interestingly enough, I bumped into the same problem with Chrome. So I've disabled all extensions that could potentially interfere, still not working. Downloaded Edge, no extensions installed: Same issue.

<img width="1552" alt="Screenshot 2023-04-09 at 02 45 22" src="https://user-images.githubusercontent.com/2648540/230748549-18fd0b1a-7711-4fbd-b5dd-43ca92a3b3cc.png">

`Get`, `Post` work fine, haven't tried `Delete` yet.

Any idea what causing this issue? Thanks.

**Reply by ganigeorgiev** [reply]
@marckrenn I don't think this is a PocketBase issue. As mentioned in https://github.com/pocketbase/pocketbase/discussions/692#discussioncomment-3780348, by default all origins are allowed (unless you've specified the `--origins` flag).

If you are behind a reverse proxy (nginx, apache, caddy, etc.) or have a node serving your frontend, double check their configuration.
